src/components/common/Page/Page.jsx

```diff
@@ -0,0 +1,18 @@
+import NavBar from '../NavBar/NavBar';
+import Footer from '../Footer/Footer';
+
+const Page = ({ children }) => {
```

Comment:
So, let's put Helmet stuff here and add props we can pass to setup

It would look like this

```suggestion
const Page = ({ children, description, title }) => {
```

Reply:
it's needed to avoid unnecessary implementation details known by `Page` component consumer

As a Page component consumer, I would prefer to set props in comparison to knowing and setting all values by myself

Reply:
For `title` we can have a default value like `title = 'Voyage'`

Or we can make it even flexible, add prop `titlePrefix = 'Voyage'` and actual `title` prop

and then inside the component we can concat them, like
```js
const pageTitle = titlePrefix + (title ? (' | ' + title) : '')
```

Reply:
Great idea. I like it